packages/mui-joy/src/Stack/Stack.tsx

```diff
@@ -0,0 +1,61 @@
+import { createStack } from '@mui/system';
+import PropTypes from 'prop-types';
+import { OverridableComponent } from '@mui/types';
+import { StackTypeMap } from './StackProps';
+import styled from '../styles/styled';
+import { useThemeProps } from '../styles';
+
+const Stack = createStack({
+  componentName: 'JoyStack',
+  createStyledComponent: styled('div', {
+    name: 'JoyStack',
+    slot: 'Root',
+    overridesResolver: (props, styles) => styles.root,
```

Comment:
We might not need this in the future because layout components do not require theming.

Reply:
I was thinking the same thing. Again, I think once we have all components in all packages, we should do a pass and fix inconsistencies/improve them.